src/modules/verifiable-credentials/verifiable-credentials-base.service.ts

```diff
@@ -436,7 +440,10 @@ export abstract class VerifiableCredentialsServiceBase {
    * @returns true if the proof is valid
    */
   public async verify<T extends ICredentialSubject>(
-    vp: VerifiablePresentation | VerifiableCredential<T>,
+    vp:
+      | VerifiablePresentation
+      | VerifiableCredential<T>
+      | StatusList2021Credential,
```

Comment:
Isn't StatusList2021Credential a specialization of VerifiableCredential for some T?